bundles/org.openhab.binding.lifx/README.md

```diff
@@ -20,7 +20,7 @@ The following table lists the thing types of the supported LIFX devices:
 | LIFX Downlight               | colorlight   |
 | LIFX GU10                    | colorlight   |
 | LIFX Mini Color              | colorlight   |
-| LIFX Tile                    | colorlight   |
+| LIFX Tile                    | tilelight    |
```

Comment:
Can you move this line below the LIFX Z?

The table is sorted by Thing Type and each group of devices having the same thing type is grouped together surrounded with a an empty row.